isa_helper.c

```diff
@@ -426,3 +424,30 @@ bool condition_check(struct CPUState curr_state, uint8_t cond)
         default: return false;
     }
 }
+
+uint32_t sign_extend(uint32_t num, uint8_t curr_width, uint8_t req_width)
```

Comment:
yah so on the remote (github) your latest commit in your branch is f68b8b6 which doesn't have header so stfu